v23.1/fips-compliance.md

```diff
@@ -0,0 +1,355 @@
+                {% comment %}Intel 64-bit Binary {% endcomment %}
+        <td>
+                    {% if r.withdrawn == true %} {% comment %} Suppress withdrawn releases. {% endcomment %}
+            <span class="badge badge-gray">Withdrawn</span>
+                    {% else %}
+            <div><a href="https://binaries.cockroachdb.com/cockroach-{{ r.release_name }}.linux-amd64.fips.tgz">Full Binary</a>{% if r.has_sha256sum == true %} (<a href="https://binaries.cockroachdb.com/cockroach-{{ r.release_name }}.linux-amd64.fips.tgz.sha256sum">SHA256</a>){% endif %}</div>
+                        {% if r.has_sql_only == true %}
+            <div><a href="https://binaries.cockroachdb.com/cockroach-sql-{{ r.release_name }}.linux-amd64.fips.tgz">SQL Shell Binary</a>{% if r.has_sha256sum == true %} (<a href="https://binaries.cockroachdb.com/cockroach-sql-{{ r.release_name }}.linux-amd64.fips.tgz.sha256sum">SHA256</a>{% endif %})</div>
+                        {% endif %}
+                    {% endif %}
+        </td>
+                {% comment %}ARM 64-bit Binary {% endcomment %}
+                {% if v_linux_arm_fips == true %}
+        <td>
+                    {% if r.withdrawn == true %} {% comment %} Suppress withdrawn releases. {% endcomment %}
+            <span class="badge badge-gray">Withdrawn</span>
+                    {% else %}
+            <div><a href="https://binaries.cockroachdb.com/cockroach-{{ r.release_name }}.linux-arm64.fips.tgz">Full Binary</a>{% if r.has_sha256sum == true %} (<a href="https://binaries.cockroachdb.com/cockroach-{{ r.release_name }}.linux-arm64.fips.tgz.sha256sum">SHA256</a>){% endif %}</div>
+                        {% if r.has_sql_only == true %}
+            <div><a href="https://binaries.cockroachdb.com/cockroach-sql-{{ r.release_name }}.linux-arm64.fips.tgz">SQL Shell Binary</a>{% if r.has_sha256sum == true %} (<a href="https://binaries.cockroachdb.com/cockroach-sql-{{ r.release_name }}.linux-arm64.fips.tgz.sha256sum">SHA256</a>{% endif %})</div>
+                        {% endif %}
+                    {% endif %}
+        </td>
+                {% endif %}
+
+                {% comment %}Docker image{% endcomment %}
+        <td>
+            <code>cockroachdb/cockroach:{{ r.release_name }}-fips</code>
+        </td>
+    </tr>
+                {% endif %}
+            {% endfor %} {% comment %}Releases {% endcomment %}
+</table>
+        {% endif %}
+    {% endfor %}{% comment %}Sections {%endcomment %}
+{% else %}
+No FIPS-ready runtimes are available at this time. Please check again later.
+{% endif %}
+
+<a id="install"></a>
+
+## Install the FIPS-ready CockroachDB runtime
+
+After you [download](#downloads) a FIPS-ready CockroachDB binary, install it in the same way as the standard binary. Refer to [Install CockroachDB on Linux](install-cockroachdb-linux.html).
+
+{{site.data.alerts.callout_info}}
+Upgrading an existing CockroachDB cluster's binary in-place to be FIPS-ready is not supported.
```

Comment:
Just checking. Is it clear or obvious what you have to do instead? Is it worth restating it here, e.g. "Instead, you need to [...]"?